src/Component/MainButton/index.js

```diff
@@ -4,7 +4,7 @@ import Button from '@mui/material/Button';
 
 export default function MainButton({ Text }) {
     return (
-        <Button variant="contained" disableElevation>
+        <Button sx={{ width: 200 }} color="secondary" variant="contained" disableElevation>
```

Comment:
This will affect responsive, can you please try to use custom styled component for example